index.html

```diff
@@ -0,0 +1,309 @@
+                <li>back end dev</li>
+                <li>
+                  <img src="./image/images-main/Counter.png" alt="counter" />
+                </li>
+                <li>2015</li>
+              </ul>
+              <p class="section-info">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="program-btns">
+                <li class="btn-ctn">
+                  <button type="submit" class="btn-html">html</button>
+                </li>
+                <li class="btn-ctn">
+                  <button type="submit" class="btn btn-css">css</button>
+                </li>
+                <li class="btn-ctn">
+                  <button type="submit" class="btn btn-js">javascript</button>
+                </li>
+              </ul>
+              <button type="submit" class="btn4">See Project</button>
+            </li>
+          </ul>
+        </section>
+        <section class="about-section">
+          <div class="about-head">
+            <h2 class="about-title">
+              About<br />
+              Myself
+            </h2>
+            <p class="about-info">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <a href="#" class="media-info">LET'S CONNECT</a>
+            <ul class="social-icons">
+              <li class="social-icon-ctn">
+                <a href="#"
+                  ><img src="./image/images-main/github.svg" alt="twitter"
+                /></a>
+              </li>
+              <li class="social-icon-ctn">
+                <a href="#"
+                  ><img src="./image/images-main/linkedin.svg" alt="linkedin"
+                /></a>
+              </li>
+              <li class="social-icon-ctn">
+                <a href="#"
+                  ><img src="./image/images-main/angellist.svg" alt="medium"
+                /></a>
+              </li>
+              <li class="social-icon-ctn">
+                <a href="#"
+                  ><img src="./image/images-main/twitter.svg" alt="github"
+                /></a>
+              </li>
+              <li class="social-icon-ctn">
+                <a href="#"
+                  ><img src="./image/images-main/medium.svg" alt="angellist"
+                /></a>
+              </li>
+            </ul>
+            <button type="submit" class="btn4">Get my resume</button>
+          </div>
+          <ul class="skills">
+            <li class="skill-head">
+              <h3 class="skill-title">
+                <span>languages</span>
+                <span
+                  ><img src="./image/images-main/arrdwn.png" alt="dropdown"
+                /></span>
+              </h3>
+
```

Comment:
- [ ] Please remove the blank line here.
Kindly keep in mind that we're only adding blank lines and indentations for keeping two logical pieces of code or two large sections apart.

> You can read about this note and other useful ones [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Reply:
Done